templates/inventario/bodegaregistro.html

```diff
@@ -0,0 +1,73 @@
+{% extends 'base.html' %}
+
+
+{% block head %}
+
+{% load static %}
+<!-- jQuery -->
+<script src="{% static 'css/registroCliente/jquery-3.3.1.slim.min.js' %}"></script>
+<!-- Fengyuan Chen's Datepicker -->
+<link rel="stylesheet" href="{% static 'css/registroCliente/datepicker.min.css' %}" />
+<script src="{% static 'css/registroCliente/datepicker.min.js' %}"></script>
+{% endblock %}
+
+{% block content %}
+<div class="container" style="background-color:#FEF0F0;">
+  <div class="row">
+    <div class="col-md-6 col-md-offset-3" style="padding: 75px">
+      <div class="panel-register panel" style="padding: 5px">
+        <div class="panel-heading" style="background-color:#FEF0F0;">
+          <div class="row mb_20">
+            <h2><center>Crear bodega</center></h2>
+          </div>
+          <hr>
+        </div>
+        <div class="panel-body" style="background-color:#FEF0F0;">
+          <div class="row">
+            <div class="col-lg-12">
+              <form id="login-form" name='crearBodega' method="post">
+                  {% csrf_token %}
+                  <div class="form-group">
```

Comment:
solo falta esto :v y approve :+1:  (en cada campo claro)
<label for="ciudad">Ciudad</label>

Reply:
que falta?